tasks/setup-Debian.yml

```diff
@@ -1,4 +1,12 @@
 ---
+
+- name: Remove OracleJDK
+  apt:
+    name: "{{ item }}"
+    state: absent
+  with_items: "{{ oracle_jdk_packages }}"
```

Comment:
`name` parameter of `apt` package handles a list, there is no need to use a loop here.